zebra-state/src/service/finalized_state/disk_db.rs

```diff
@@ -834,7 +839,19 @@ impl DiskDb {
             .map(|cf_name| rocksdb::ColumnFamilyDescriptor::new(cf_name, db_options.clone()));
 
         let db_result = if read_only {
-            DB::open_cf_descriptors_read_only(&db_options, &path, column_families, false)
+            // TODO: Make this path configurable?
+            let secondary_path =
+                config.db_path("secondary_state", format_version_in_code.major, network);
```

Comment:
Should this be a tempfile instead? Is it okay to always open a secondary db instance instead of a read-only one whenever `read_only` is set to true, or do we want to still be able to open a read-only db? I only see a read-only db being used from a `zebra-utils` binary for reading scanning results, where it seems fine to always use a secondary db instance instead.

Reply:
Started using a tempfile in https://github.com/ZcashFoundation/zebra/pull/8596/commits/15feed173daced582340fdcdb4d4bb8cc9593a39.

It seems fine to use a secondary db instance instead of a read-only instance when `read_only` is true.